AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 rshb.ru###alert.bottom
+autohero.com##header > div > [class^="root"]
```

Comment:
Something like that `autohero.com##header > div[class^="container_"] > div[class^="body_"] + div[class^="root_"]` will be more accurate.

Reply:
Okay. I will check again.

Reply:
@zloyden How to use `~` and `+`?

Reply:
There is a useful [information ](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors)about it.

Reply:
@zloyden I still don't understand :( it doesn't have pictures/diagrams to explain properly.

Reply:
Just imagine a tree. You choose a branch with 10 leaves in it. Let's say you need to take only the third sheet. Select the previous element and specify the third one, this is a kind of additional condition. The result is 2+3 `div[class^="body_"] + div[class^="root_"]`. But if this doesn't work, you can change the condition to a more previous element, but the result will be 1~3 `div.someclassname ~ div[class^="root_"]`.

Reply:
Thanks @zloyden. Actually I am a Software Engineer, who is on vacation, so I don't really focus on this (CSS Selectors). I am not disagreeing that I didn't learn it, as I learned it when I was doing my course in University. Mainly I focus on C++, so I might forget all these.